emulator/src/components/hdw-tft/hdw-tft.c

```diff
@@ -13,7 +13,7 @@
 // Const variables
 //==============================================================================
 
-static const uint32_t paletteColorsEmu[216] = {
+static const uint32_t paletteColorsEmu[217] = {
```

Comment:
What was this added for, transparency? If you're gonna increase the size of the array, you should add an element to it too.

Reply:
Undid this, fixed the culprit